ixofoundation/ixo-webclient
# chore: update control panel text

## Scope
https://nonaredteam.atlassian.net/browse/IXO-748

## Work Done
Updated the text
Updated the control panel scroll position when fuelling an entity

## Steps to Test
Check that the text is updated on the UI
Check that the control panel is scrolled to the top when you click fuel an entity
